src/lib/Repository/UiUserService.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\UserService;
+
+/**
+ * Service for loading user.
+ */
+class UiUserService
+{
+    /**
+     * @var UserService
+     */
+    private $userService;
+
+    /**
+     * @var Repository
+     */
+    private $repository;
+
+    public function __construct(Repository $repository, UserService $userService)
+    {
+        $this->repository = $repository;
+        $this->userService = $userService;
+    }
+
+    /**
+     * Loads a user ignoring privileges.
+     *
+     * @param mixed $userId
+     *
+     * @return \eZ\Publish\API\Repository\Values\User\User
```

Comment:
phpdoc: This may return `null `. Also, the method's description should be updated to reflect the `find` approach (as opposed to `load`).